infrastructure/prometheus/prometheus.yml

```
@@ -0,0 +1,54 @@
+  # Redis
+  - job_name: 'redis'
+    static_configs:
+      - targets: ['redis:6379']
+    scrape_interval: 30s
+
+  # LocalStack
+  - job_name: 'localstack'
+    static_configs:
+      - targets: ['localstack:4566']
+    metrics_path: '/_localstack/health'
+    scrape_interval: 60s 
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)